admin/class-suggested-plugins.php

```diff
@@ -109,16 +109,17 @@ protected function get_yoast_seo_suggested_plugins_notification( $name, $plugin,
 	 * @return string The install suggested plugin message.
 	 */
 	protected function create_install_suggested_plugin_message( $suggested_plugin, $third_party_plugin ) {
```

Comment:
The second argument is not used anymore, so it can be removed.

It can also be removed from the calls to this method.
```suggestion
	protected function create_install_suggested_plugin_message( $suggested_plugin ) {
```